audit/projects/k8s-infra-e2e-boskos-scale-03/buckets/artifacts.k8s-infra-e2e-boskos-scale-03.appspot.com/iam.json

```diff
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "projectEditor:k8s-infra-e2e-boskos-scale-03",
+        "projectOwner:k8s-infra-e2e-boskos-scale-03"
+      ],
+      "role": "roles/storage.legacyBucketOwner"
+    },
+    {
+      "members": [
+        "projectViewer:k8s-infra-e2e-boskos-scale-03"
+      ],
+      "role": "roles/storage.legacyBucketReader"
+    }
+  ]
+}
```

Comment:
It looks like GCR buckets are getting auto-created by CI if they don't already exist.  

We pre-create these buckets for staging projects, it may be worth considering doing the same for e2e projects:
- we would avoid this ongoing noise as e2e projects eventually get the bucket added if they don't already have it
- the IAM permissions are likely slightly different compared to GCR buckets in staging projects

Reply:
Opened https://github.com/kubernetes/k8s.io/issues/1999 to track this as a followup